library/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-api:$junitVersion"
     testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junitVersion"
     testImplementation "org.junit.jupiter:junit-jupiter-params:$junitVersion"
+    testImplementation "org.mockito:mockito-core:$mockitoVersion"
```

Comment:
There's another lib I use in Kotlin development to replace Mockito -> MockK.
I prefer its DSL to right Kotlin tests.
Try it and tell me what you think about.

BTW it won't block the PR

Reply:
Yup let's give it a try :)